dlpy/model.py

```diff
@@ -106,7 +111,7 @@ def fit(self, data, inputs=None, target=None, data_specs=None, mini_batch_size=1
             lr=0.01, optimizer=None, nominals=None, texts=None, target_sequence=None, sequence=None, text_parms=None,
             valid_table=None, valid_freq=1, gpu=None, attributes=None, weight=None, seed=0, record_seed=0,
             missing='mean', target_missing='mean', repeat_weight_table=False, force_equal_padding=None,
-            save_best_weights=False, n_threads=None, target_order='ascending', train_from_scratch=None):
+            save_best_weights=False, n_threads=None, target_order='ascending', train_from_scratch=None, tensorboard=None):
```

Comment:
please remove this from here. and create another api having this one. check out fit_and_visualize

Reply:
fit_tensorboard() created